apps/okreads-e2e/src/specs/reading-list.spec.ts

```diff
@@ -17,4 +17,31 @@ describe('When: I use the reading list feature', () => {
       )
     );
   });
+
+  it('Then: I should see mark book as finished', async () => {
+    await browser.get('/');
+    await browser.wait(
+      ExpectedConditions.textToBePresentInElement($('tmo-root'), 'okreads')
+    );
+
+    // search for books
+    const form = await $('form');
+    const input = await $('input[type="search"]');
+    await input.sendKeys('react');
+    await form.submit();
+
+    // add book to reading list
+    await $('[data-testing="add-to-reading-list"]').click();
```

Comment:
As in Task 3, this throws a warning - better to use the element array finder and select the first enabled button.

Reply:
Updated same as in task 3